### PR TITLE
fix 'conect' typo

### PR DESCRIPTION
**Pull request by IsQiao**
**PR Title**
 Fix "conect" typo

**PR Description**
* Fix "conect" typo in solution

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/aspnetcore?pullRequest=30558) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/aspnetcore?pullRequest=30558) before we can accept your contribution.<br/><br/>:x: IsQiao [sign now](https://cla.dotnetfoundation.org/dotnet/aspnetcore?pullRequest=30558)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/aspnetcore?pullRequest=30558) it.</sub>

File: src/SignalR/common/Http.Connections/src/ConnectionOptionsSetup.cs [hunk 1]
```diff
@@ -14,7 +14,7 @@ public class ConnectionOptionsSetup : IConfigureOptions<ConnectionOptions>
         /// <summary>
         /// Default timeout value for disconnecting idle connections.
         /// </summary>
-        public static TimeSpan DefaultDisconectTimeout = TimeSpan.FromSeconds(15);
+        public static TimeSpan DefaultDisconnectTimeout = TimeSpan.FromSeconds(15);
```
Comment (BrennanConroy) [inline src/SignalR/common/Http.Connections/src/ConnectionOptionsSetup.cs:17]: We've previously discussed changing this and decided it's not worth making a breaking change here.
https://github.com/dotnet/aspnetcore/pull/18494

File: src/SignalR/common/Http.Connections/src/Internal/HttpConnectionDispatcher.Log.cs [hunk 2]
```diff
@@ -44,7 +44,7 @@ internal static class Log
                 LoggerMessage.Define<HttpTransportType>(LogLevel.Trace, new EventId(11, "ReceivedDeleteRequestForUnsupportedTransport"), "Received DELETE request for unsupported transport: {TransportType}.");
 
             private static readonly Action<ILogger, Exception?> _terminatingConnection =
-                LoggerMessage.Define(LogLevel.Trace, new EventId(12, "TerminatingConection"), "Terminating Long Polling connection due to a DELETE request.");
+                LoggerMessage.Define(LogLevel.Trace, new EventId(12, "TerminatingConnection"), "Terminating Long Polling connection due to a DELETE request.");
```
Comment (BrennanConroy) [inline src/SignalR/common/Http.Connections/src/Internal/HttpConnectionDispatcher.Log.cs:47]: This was also discussed in https://github.com/dotnet/aspnetcore/pull/18494 and not taken as the EventId is considered part of the API.

**Review by BrennanConroy — COMMENTED** [review]
I think we should just add comments on both typos saying they wont be fixed as they are breaking changes and not important enough to make a breaking change.

That will hopefully help people avoid opening PRs for the typos in the future 😄

**Pull request closed without merging**